front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1699,6 +1711,46 @@ export type GetV2TimetableByIdConflictsApiArg = {
   id: number;
   infraId: number;
 };
+export type PostV2TimetableByIdStdcmApiResponse =
+  /** status 201 The simulation result */
+  | {
+      departure_time: string;
+      path: PathfindingResultSuccess;
+      simulation: SimulationResponse;
+      status: 'success';
+    }
+  | {
+      status: 'path_not_found';
+    };
+export type PostV2TimetableByIdStdcmApiArg = {
+  /** The infra id */
+  infra: number;
+  /** timetable_id */
+  id: number;
+  body: {
+    comfort: Comfort;
+    margin?: MarginValue | null;
+    /** By how long we can shift the departure time in milliseconds */
+    maximum_departure_delay?: number;
+    /** Specifies how long the total run time can be in milliseconds */
+    maximum_run_time?: number;
+    rolling_stock_id: number;
+    /** Train categories for speed limits */
+    speed_limit_tags?: string | null;
+    start_time: string;
```

Comment:
Why are we using `start_time` here but `maximum_departue_delay` above ?

Reply:
maximum_departure_delay refers to the delay you allow the train when it departs. 
Let's say you have a start_time`` today of 8:00 and the `maximum_departure_delay` is 5 min. This means you allow your train to leave at 8:05 at the latest.

Reply:
Yes but I mean, why not use `start_time` and `maximum_start_time` or `departure_time` and `maximum_departure_delay` instead of mixing both ?

Reply:
We can leave it like that its fine